docs/src/main/asciidoc/_admin-guide/management-tasks/Configuration_file_git_history.adoc

```diff
@@ -124,3 +124,130 @@ You may need to push your repository changes to a remote repository so you can s
 [standalone@localhost:9990 /] :publish-configuration(location="origin")
 {"outcome" => "success"}
 ----
+
+[[ssh_auth]]
+== SSH Authentication
+
+In order to connect to any SSH git server to manage your configuration file history, you must use an Elytron configuration
+file to specify your SSH credentials. There are a number of ways to specify these credentials:
+
+=== SSH Key Location Credential
+
+It is possible to reference a file containing your SSH keys as follows:
+
+[source,xml,options="nowrap"]
+----
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+    <authentication-client xmlns="urn:elytron:client:1.5">
+        <authentication-rules>
+            <rule use-configuration="test-login">
+            </rule>
+        </authentication-rules>
+        <authentication-configurations>
+            <configuration name="test-login">
+                <credentials>
+                    <ssh-private-key ssh-directory="/home/user/git-persistence/" private-key-file="id_ec_test" known-hosts-file="known_hosts">
+                        <clear-password password="secret"/>
+                    </ssh-private-key>
+                </credentials>
+            </configuration>
+        </authentication-configurations>
+    </authentication-client>
+</configuration>
+----
+This configuration indicates that the private key to be used for SSH authentication is in the file `id_ec_test` in the
+directory `/home/user/git-persistence` and the passphrase "secret" is needed to decrypt the key.
+
+The `ssh-private-key-` credential also take the following attributes:
+* `known-hosts-file` - this is to specify the name of the file containing the known SSH hosts you trust. This file should
```

Comment:
I think check the formatting here so this gets rendered properly as a bullet. It would be good to have a bulleted list of all the attributes and then a bulleted list of the possible child elements.

Reply:
Fixed, thanks!